.github/workflows/dockers-agent-sidecar-image.yaml

```
@@ -298,4 +298,15 @@ jobs:
     uses: ./.github/workflows/_docker-image.yaml
     with:
       target: agent-sidecar
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 301: could not find expected ':'

(syntax-check)